dbt/models/staging/stg_organisations.sql

```diff
@@ -10,6 +10,11 @@ select
     appartenance_geo_communes.nom_zone_emploi                                  as zone_emploi,
     appartenance_geo_communes.nom_epci                                         as epci,
     organisations_libelles.libelle                                             as type_complet,
+    case
+        when organisations.type = 'ML' or organisations.type = 'PE' or organisations.type = 'CAP_EMPLOI' then 'SPE'
```

Comment:
```suggestion
        when organisations.type in ('ML', 'PE', 'CAP_EMPLOI') then 'SPE'
```
serait plus élégant (DNRY) et devrait fonctionner non ?